objects/isn_sharedobjectscripts.lua

```diff
@@ -31,18 +31,18 @@ function isn_getAllDevicesConnectedOnNode(node,direction)
 end
 
 function isn_numericRange(number,minimum,maximum)
-	if number > maximum then return maximum end
-	if number < minimum then return minimum end
-	return number
+	if number > maximum then 
+    return maximum
+	elseif number < minimum then 
+    return minimum
+  else
+    return number
+  end
```

Comment:
`isn_getListLength` is intended to get the list length.  `#`gets the list length.  I wasn't going to mention this but actually there is a difference which makes `#arg` better where outcome is concerned.  The existing function is flawed.  Your current usage doesn't trigger the flaw, but for example lets say someone did this

```
local myTable = {12321, 123123, 3.14, "string", {"Chuckle Sushi"} }
myTable.not_a_number = true
local mySize = isn_getListLength(myTable)
local i = 1
while (i <= mySize) do
  sb.logInfo( tostring(myTable[i]) )
  i = i + 1
end
```

That's all very typical code.  Here `isn_getListLength` will return `6` instead of `5`.  You'd end up referencing a nil index because `pairs()` will use non-numeric indexes as well as numeric.  For some things that's desirable but getting a list length isn't one of them.  You could change that to` ipairs()` to correct this but `#args` is just better.

Now this is only currently used in _objects\power\isn_radiostation\isn_radiostation.lua_ and only on an iterated list (I checked with notepad++'s "find in files") and as long it is never used any other way... I guess you'll be okay.

I do not understand the reason for your objection to changing it.  If you feel like I'm stepping on your feet... I'm sorry.  Just trying to be helpful.